### PR TITLE
A/b testing

### PR DESCRIPTION
**Pull request by yelim237**
Ask for merge 'A/b testing' (project 1) to 'develop'

### CODE REVIEW

File: 120722_photo/RGBv2np.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import numpy as np
+import cv2
+from PIL import Image
```
Comment (hsmoon5458) [inline 120722_photo/RGBv2np.py:3]: what is it

File: 120722_photo/RGBv2np.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    image = Image.open(x) 
+    height = image.get_height()
+    width = image.get_width()
+    new_image = Image(height, width)
+
+    for r in range(height):
+        for c in range(width):
+            temp = image.get_pixel(r, c)
+            temp2 = new_image.get_pixel(r, c)
```
Comment (hsmoon5458) [inline 120722_photo/RGBv2np.py:15]: temp2 is not being used anyhwere, so you can remove it

File: 120722_photo/RGBv2np.py (hunk 1 above)
Comment (yelim237) [reply 120722_photo/RGBv2np.py:3]: PIL stands for Python Imaging Library which is an imaging processing package.

File: 121322_ABtesting/ABtesting.py [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+control_data.columns = ["Campaign Name", "Date", "Amount Spent", "Number of Impressions", "Reach", "Website Clicks", "Searches Received", "Content Viewed", "Added to Cart", "Purchases"]
+test_data.columns = ["Campaign Name", "Date", "Amount Spent", "Number of Impressions", "Reach", "Website Clicks", "Searches Received", "Content Viewed", "Added to Cart", "Purchases"]
+
+print(control_data.isnull().sum())
+print(test_data.isnull().sum())
+# Q: The column "Reach" not appeared in the list, even if the number of columns and the list of the columns are the same
+# print(control_data["Reach"].isnull().sum())
+
+# 'inplace' indicates to make changes in the original data frame itself if True
```
Comment (hsmoon5458) [inline 121322_ABtesting/ABtesting.py:28]: do not copy the entire code

**Review by hsmoon5458 — APPROVED** [review]

**Pull request merged**